core/storage/trie/polkadot_trie/polkadot_trie_cursor.cpp

```diff
@@ -240,12 +233,9 @@ namespace kagome::storage::trie {
   }
 
   auto PolkadotTrieCursor::constructLastVisitedChildPath(
-      const common::Buffer &key)
-      -> outcome::result<
-          std::list<TriePathEntry>> {
+      const common::Buffer &key) -> outcome::result<std::list<TriePathEntry>> {
```

Comment:
Minor. Think about to replace type of result inference. It's would be more readable.